babel.config.js

```diff
@@ -29,6 +29,45 @@ module.exports = {
          * if you need console in production use ONLY console.error or console.warn
          */
         ["transform-remove-console", { exclude: ["error", "warn"] }],
+
+        /**
+         * Transforms member style imports into default style imports
+         * (import {x} from 'y') into (import x from 'y/lib/x')
+         */
```

Comment:
I think this is a Good approach for optimisation , what do do think @christin-wednesday ?

Reply:
🚀